src/main/cpp/subsystems/SwerveDrive.cpp

```diff
@@ -56,53 +56,6 @@ bool SwerveDrive::Init() {
     OKC_CHECK(right_front_module_ != nullptr);
     OKC_CHECK(right_back_module_ != nullptr);
 
-    // create location objects
-    left_front_loc_ = std::make_shared<frc::Translation2d>();
-    left_back_loc_ = std::make_shared<frc::Translation2d>();
-    right_front_loc_ = std::make_shared<frc::Translation2d>();
-    right_back_loc_ = std::make_shared<frc::Translation2d>();
-
-    OKC_CALL(left_front_module_->GetLocationOnRobot(left_front_loc_.get()));
-    OKC_CALL(left_back_module_->GetLocationOnRobot(left_back_loc_.get()));
-    OKC_CALL(right_front_module_->GetLocationOnRobot(right_front_loc_.get()));
-    OKC_CALL(right_back_module_->GetLocationOnRobot(right_back_loc_.get()));
-    
-    OKC_CHECK(left_front_loc_ != nullptr);
-    OKC_CHECK(left_back_loc_ != nullptr);
-    OKC_CHECK(right_front_loc_ != nullptr);
-    OKC_CHECK(right_back_loc_ != nullptr);
-
-    // define SwerveKinematics object
-    swerve_kinematics_ = std::make_shared<frc::SwerveDriveKinematics<4>>(*left_front_loc_, *left_back_loc_, *right_front_loc_, *right_back_loc_);
-    OKC_CHECK(swerve_kinematics_ != nullptr);
-
-    // create position objects
-    left_front_pos_ = std::make_shared<frc::SwerveModulePosition>();
-    left_back_pos_ = std::make_shared<frc::SwerveModulePosition>();
```

Comment:
Is all this deleted stuff important? If not, carry on lol

Reply:
shouldn't be important, that was also part of the WPILib swerve stuff. I think removing it is fine, as the current custom tele-op code works fine without it.